### ISSUE TITLE
1.63g Fire On Release doesn't work anymore

### ISSUE THREAD

**Issue by Gui205** [suriyun-production/mmorpg-kit-docs#882]
As the title say in 1.63g the bow when he's setup as Fire on release mode doesn't work anymore, he stay stuck on isWeaponcharge state and that's block all ( That mean if you unequip and equip another weapon the weapon will not work too )

**Comment by insthync** [comment]
Fixed in 1.63h, for quick fix, change codes in `BaseCharacterEntity` -> `CanDoActions` to
```
        public virtual bool CanDoActions()
        {
            return !this.IsDead() && !IsAttackingOrUsingSkill && !IsReloading && !IsPlayingActionAnimation();
        }
```

**Comment by Gui205** [comment]
> Fixed in 1.63h, for quick fix, change codes in `BaseCharacterEntity` -> `CanDoActions` to
> 
> ```
>         public virtual bool CanDoActions()
>         {
>             return !this.IsDead() && !IsAttackingOrUsingSkill && !IsReloading && !IsPlayingActionAnimation();
>         }
> ```

Work nice !
